### DISCUSSION TITLE
Permission to reuse  part of figure 1 from prostate cancer paper for thesis

### DISCUSSION THREAD

**Discussion by mxdeluca** [ShixiangWang/self-study#37] [Q&A] [answered]
Hi,
I am writting my PhD thesis in the area of breast cancer genomics and would like to use part of figure 1 from your prostate cancer paper in my thesis:
![image](https://user-images.githubusercontent.com/37563357/98877392-205fa700-24cc-11eb-997c-65eb46d16fe8.png)
Would you grant me persmission?
Of course, full attribution will be provided.
Regards
Marc

**Comment by ShixiangWang** [comment, marked as answer]
@mxdeluca Hello Marc,

I would like to grant your permission if you cite this paper properly.

Best,

Shixiang

**Comment by mxdeluca** [comment]
Shall do! Thank you Shixiang !
